### ISSUE TITLE
Add note in README about upstreaming to kernel 6.1

### ISSUE THREAD

**Issue by h4emp3** [zeule/asus-ec-sensors#32]
Hey there, I'm happily using this kernel module for a while now, very happy with it!

When I was updating to kernel 6.1 just now, dkms gave me an error telling me, that the module does already exist. So I guess you got this upstreamed? Congratulations!

I think it would be nice if this was readily appearent when reading the readme of this project though :)

**Comment by zeule** [comment]
Thank you, apparently the message in the 3r paragraph of the readme is missable. Any suggestions?

**Comment by h4emp3** [comment]
Well, damn. I guess my attention span isn't as great as I hoped :D  The paragraph is completely sufficient.

For what it's worth: I've searched for "upstream" and "6.1". But appearently 6.1 is not the point where this module was upstreamed, debian just did not prebuild/package it before this version. I did not even think about that possibility. And I also searched for some of the output dkms gives you when trying to build a kernel module that already exists.

Thanks for the quick reply and sorry for only skimming the readme and taking up your time because of it!
